softwareComponents/configuration/test/rofibot.cpp

```diff
@@ -34,13 +34,13 @@ TEST_CASE( "Universal Module Test" ) {
         auto um = UniversalModule( 0, 0_deg, 0_deg, 0_deg );
         CHECK( um.connectors().size() == 6 );
 
-        for ( int i = 0; i < um.connectors().size(); i++ ) {
+        for ( auto i = 0; i < um.connectors().size(); i++ ) {
```

Comment:
Use `size_t` 0 `auto` here is a fancy way of saying `int` (you will still get `-Wconversion` on the comparison).